src/bbtag/utils/json.ts

```diff
@@ -85,8 +85,6 @@ export const json = Object.freeze({
     getPathKeys(path: string | readonly string[]): readonly string[] {
         if (typeof path !== 'string')
             return path;
-        if (path === '')
-            return [];
```

Comment:
What happens when you do `getPathKeys("");` ?

Reply:
you get back `[""]` now, which means "go to the empty key"